README.md

```diff
@@ -0,0 +1,49 @@
+# Tkey Rust Android Example
+
+This project is an example Android application demonstrating the usage of the Tkey Rust library in an Android environment. Tkey Rust is a library that provides cryptographic functionality and key management for decentralized applications (dApps) built on the Torus network.
+
+## Getting Started
+
+To get started with this project, follow the steps below:
+
+### Prerequisites
+
+- Android Studio: Install the latest version of Android Studio, which is the official Integrated Development Environment (IDE) for Android development.
+
+### Installation
+
+1. Clone the repository to your local machine:
+
+   ```shell
+   git clone https://github.com/torusresearch/tkey-rust-android-example.git
+   ```
+2. Open Android Studio and select "Open an existing Android Studio project."
+
+3. Navigate to the cloned directory and select the tkey-rust-android-example folder.
+
+4. Let Android Studio build the project and download any necessary dependencies.
+
+### Usage
+Connect your Android device to your development machine or use an emulator.
+
+In Android Studio, click on the "Run" button or use the shortcut Shift + F10 to build and run the application.
+
+The application will be installed and launched on your device or emulator. You can now explore the functionality of the Tkey Rust library in the app.
+
+### Customization
+The example project provides a basic implementation of using the Tkey Rust library. You can customize and extend it according to your needs. Here are a few suggestions:
+
+Modify the UI: Update the user interface to match your application's design and requirements.
+
+Implement additional functionality: Extend the example project to include more features and interactions with the Tkey Rust library.
+
+Integrate with other libraries or APIs: If your application requires integration with other libraries or APIs, you can incorporate them into this project.
+
+### Contributing
+Contributions to this project are welcome. If you have any suggestions, bug fixes, or improvements, please open an issue or submit a pull request.
+
+### License
+This project is licensed under the MIT License. You are free to modify and distribute the codebase following the terms of the license.
+
+### Resources
+Tkey Rust GitHub Repository: https://github.com/torusresearch/tkey-rust (Private)
```

Comment:
This is a private link, it can help web3auth employees, should I remove or keep ?

Reply:
Remove it, inaccessible resources should not be included.

Reply:
Rather add the references to the private repository.